source/includes/_event.md

```diff
@@ -187,19 +187,19 @@ Get the details and dates of a specific event.
 
 
 ```shell
-curl -X POST 'https://api.inviplay.nl/event/subscribe/user' \
+curl -X PUT 'https://api.inviplay.nl/event/subscribe/user' \
```

Comment:
deze url moet hetzelfde zijn als op regel 232 toch? (klopte dus ook al niet)

Reply:
Klopt, heb 'm aangepast. Op de een of andere mnier zie je dat niet in de PR....